src/DeferredFutures.jl

```diff
@@ -47,6 +47,21 @@ function Base.show(io::IO, ref::DeferredFuture)
     print(io, "$(typeof(ref).name.name) at ($(rc.where),$(rc.whence),$(rc.id))")
 end
 
+"""
+    serialize(s::AbstractSerializer, ref::DeferredFuture)
+
+Serialize a DeferredFuture such that it can de deserialized by `deserialize` in a cluster.
+"""
+function Base.serialize(s::AbstractSerializer, ref::DeferredFuture)
+    if VERSION < v"0.6.0-rc1"
```

Comment:
Do you have any suggestions on how to get this right on the first try? This has been something I have struggled with, and GitHub doesn't make it easy either.